docs/pages/docs/getting-started.mdx

```diff
@@ -109,7 +109,7 @@ The `turbo` package is a little shell that will install the proper `turbo-<os>-<
 
 In the root of your monorepo, create an empty file named `turbo.json`. This will hold the configuration for Turborepo.
 
-If your git repo's base branch is not `origin/master`, then you need to specify a `baseBranch` too. For example, `origin/main`.
+If your git repo's base branch is not `origin/main`, then you need to specify a `baseBranch` too. For example, `origin/main`.
```

Comment:
The example should be updated to show non default behavior now that `baseBranch` defaults to `origin/main`

Reply:
🤦🏾  good catch, updated